n2y/blocks.py

```diff
@@ -582,8 +582,13 @@ def __init__(self, client, notion_data, page, get_children=True):
     def to_pandoc(self):
         # TODO: in the future, if we are exporting the linked page too, then add
         # a link to the page. For now, we just display the text of the page.
+        if self.link_type == "page_id":
```

Comment:
Is there a reason we're inlining this logic?

Reply:
I had thought that, since we know what type of link it is, it would be better to avoid the unnecessary API request that's present if we use `get_page_or_database`.